2018q4/bcm58712.md

```diff
@@ -0,0 +1,11 @@
+## Broadcom ARM64 SoC support ##
+
+Contact: Michal Stanek, <mst@semihalf.com>, Marcin Wojtas, <mw@semihalf.com>
+
+Semihalf has recently started work on FreeBSD support for BCM5871X SoC series. These are quad-core 64-bit ARMv8 Cortex-A57 communication processors targeted for
```

Comment:
Perhaps a link to https://www.broadcom.com/products/embedded-and-networking-processors/communications/bcm58712/ ?

Reply:
Hi Ed, thanks for your comments - link added.